pkg/restapi/device/operations.go

```diff
@@ -24,16 +25,16 @@ import (
 
 // Endpoints.
 const (
-	registerBeginPath  = "register/begin"
-	registerFinishPath = "register/finish"
-	loginBeginPath     = "login/begin"
-	loginFinishPath    = "login/finish"
+	registerBeginPath  = "/register/begin"
```

Comment:
[`deviceBasePath`](https://github.com/trustbloc/edge-agent/blob/ffd33ea110d37b9d129340d1420bd5e2496553f4/cmd/http-server/startcmd/start.go#L152) already has a trailing slash... no need to add it here

Reply:
not adding trailing slash gives 404 page not found error. I noticed even 	oidcLoginPath    = "/login" starts with training slash. It doesnt work without that

Reply:
ok